modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/messaging/CheckRedirectNotFoundEntriesMessageListener.java

```diff
@@ -47,7 +46,7 @@
  */
 @Component(
 	configurationPid = "com.liferay.redirect.internal.configuration.RedirectConfiguration",
-	service = MessageListener.class
+	service = {}
```

Comment:
@yuhai You say `The commit will use _schedulerEngineHelper.register() to register MessageListener service.` but I didn't see this change?

Reply:
@yuhai Oh, it's already there. Then you should change your commit message.